src/main/kotlin/graphics/scenery/volumes/DisplayRangeEditor.kt

```diff
@@ -0,0 +1,92 @@
+package graphics.scenery.volumes
+
+import graphics.scenery.ui.RangeSlider
+import net.miginfocom.swing.MigLayout
+import javax.swing.*
+import kotlin.math.max
+import kotlin.math.roundToInt
+
+/**
+ * GUI for editing the display range of volumes.
+ * Part of TransferFunctionEditor
+ */
+class DisplayRangeEditor(private val tfContainer: HasTransferFunction): JPanel(){
+
+    //RangeEditor
+    private val minText: JTextField
+    private val maxText: JTextField
+    private val rangeSlider: RangeSlider
+    private val minValueLabel: JLabel
+    private val maxValueLabel: JLabel
+
+    init {
+        this.layout = MigLayout(
```

Comment:
Unnecessary `this`.

Reply:
In addition, MigLayout's constraints are not exactly intuitive. I just spend some time yesterday with them 😂 Could you maybe add a comment that clarifies what those are intended to achieve?

Reply:
You are completly right. I also have no Idea what they do. I just copied them from here ;) https://github.com/scenerygraphics/scenery/blame/2fc22df0e998fc8c0f641cacd993a2e707f61fcc/src/main/kotlin/graphics/scenery/volumes/TransferFunctionEditor.kt#L344